docs/UserGuide.md

```diff
@@ -194,6 +195,19 @@ You have successfully removed:
 __________________________________________
 ```
 
+<ins>If there is only one item with a name containing or with different upper or lower case to `ITEM_NAME`</ins>:
```

Comment:
Could this be changed to "If there is only one item with a similar spelling (different capitalization / has more letters)"?

Reply:
done